onnxruntime/core/providers/cpu/nn/qlinearconv.cc

```diff
@@ -518,11 +503,47 @@ Status QLinearConv<int8_t>::Compute(OpKernelContext* context) const {
       ReorderFilter(static_cast<const uint8_t*>(W->DataRaw()),
                     reordered_W,
                     static_cast<size_t>(M),
-                    static_cast<size_t>(group_input_channels),
+                    static_cast<size_t>(W_shape[1]),
                     static_cast<size_t>(kernel_size));
+      is_W_signed = W->IsDataType<int8_t>();
     }
   }
 
+  int64_t group_count = conv_attrs_.group;
+  int64_t group_input_channels = W_shape[1];
+  int64_t group_output_channels = M / group_count;
+
+  // Test for depthwise convolution.
+  const bool is_depthwise_conv = (use_reordered_W && group_input_channels == 1 && group_output_channels == 1);
+  if (is_depthwise_conv) {
+    // Update the input and output channels to the number of groups in order to
+    // reuse as much of the below standard convolution path.
+    group_input_channels = group_count;
+    group_output_channels = group_count;
+    group_count = 1;
```

Comment:
>group_count [](start = 4, length = 11)

it takes me time to understand the code below until I see this line

Reply:
I was thinking of renaming it to inner_input_channels but held back. If that would make it clearer, I can do that in a subsequent change.